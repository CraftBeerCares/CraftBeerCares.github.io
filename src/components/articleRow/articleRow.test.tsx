import { render } from "@testing-library/react";
import ArticleRow from "./articleRow";

describe("renders ArticleRow", () => {
  it.each`
  imageFirst
  ${true}
  ${false}
`("imageFirst=$imageFirst", async ({ imageFirst }) => {
  const { container } = render(<ArticleRow imageSrc="/assets/001.webp" imageAlt="test alt" imageFirst={imageFirst}><p>Hello</p></ArticleRow>);
  expect(container).toMatchSnapshot();
});
})

