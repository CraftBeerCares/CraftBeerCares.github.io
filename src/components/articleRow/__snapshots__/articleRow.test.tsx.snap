// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ArticleRow imagePosition=left 1`] = `
<div>
  <div
    class="articleRow"
  >
    <div
      class="articleImage imageLeft"
    >
      <img
        alt="test alt"
        src="/assets/001.webp"
        style="object-fit: contain; width: 100%; margin-left: -3rem;"
      />
    </div>
    <article
      class="article articleRight"
    >
      <p>
        Hello
      </p>
    </article>
  </div>
</div>
`;

exports[`renders ArticleRow imagePosition=right 1`] = `
<div>
  <div
    class="articleRow"
  >
    <div
      class="articleImage imageLeft"
    >
      <img
        alt="test alt"
        src="/assets/001.webp"
        style="object-fit: contain; width: 100%; margin-left: -3rem;"
      />
    </div>
    <article
      class="article articleRight"
    >
      <p>
        Hello
      </p>
    </article>
  </div>
</div>
`;
